Cargo.toml

```diff
@@ -58,3 +58,9 @@ members = [ "stress" ]
 name = "bench_recovery"
 harness = false
 required-features = ["failpoints"]
+
+[[test]]
```

Comment:
You should send another PR to move all existing failpoint tests to this folder.

Reply:
OK. I'd like to move io error test to this folder first in this PR, then send another one to move the existing test.